src/react-components/src/components/01-atoms/Images/Icons/arrow.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="14" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M10.526 12.871L.263 1.676 1.737.324 12 11.52 22.263.324l1.474 1.352L13.474 12.87a2 2 0 01-2.948 0z"/></svg>
```

Comment:
I think one of these things was replaced in Template - note to self to not overwrite that work

Reply:
I'll handle the merge conflicts, you don't have to worry about it